vsts.yml

```diff
@@ -13,10 +13,15 @@ jobs:
       export PATH="$PATH:/Users/electron/depot_tools"
       echo "##vso[task.setvariable variable=PATH]$PATH"
       export GIT_CACHE_PATH="/Users/electron/libcc_cache"
+      if [ "ELECTRON_RELEASE" != "" ]; then
```

Comment:
Should be `if [ "$ELECTRON_RELEASE" == "1" ]; then`

Reply:
Is there a different between an empty var check and exact value of `1`  check?
Oh, I even forgot the `$` sign )

Reply:
Fixed.